core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5335,6 +5335,14 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.IGNORE)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey FUSE_JNIFUSE_LIBFUSE_VERSION =
+      new Builder(Name.FUSE_JNIFUSE_LIBFUSE_VERSION)
+          .setDefaultValue(0)
```

Comment:
I think if the user has explicitly set the version, the user must care about the exact version that is going to be used. We should throw an error instead of loading another version if it cannot be loaded. This tells the user there might be some configuration or environment issues and the user can then go investigating it.

It is confusing to use other values here. How about only accept 2, 3 and 0 (for automatic choosing) and throw if it's not in the range?

Reply:
This is exactly the current design. Current default value is 0, which is 2 then 3, and the user can set to 2, 3 if they want a specific version.